app/javascript/spec/workflow-repository-form/workflow-repository-form.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import fetchMock from 'fetch-mock';
+import { shallow } from 'enzyme';
+import WorkflowRepositoryForm from '../../components/workflow-repository-form';
+
+require('../helpers/set_fixtures_helper.js');
+require('../helpers/old_js_file_require_helper.js');
+require('../helpers/miqSparkle.js');
+require('../helpers/miqAjaxButton.js');
+
+jest.mock('../../helpers/miq-redirect-back', () => jest.fn());
+
+describe('Ansible Repository form component', () => {
+  let submitSpyMiqSparkleOn;
+  let submitSpyMiqSparkleOff;
+
+  const attributes = ['name', 'description', 'scm_url', 'verify_ssl', 'authentication_id', 'scm_branch'];
+
+  const repositoryMock = {
+    authentication_id: null,
+    description: 'playbooks for testing',
+    href: 'http://localhost:3000/api/configuration_script_sources/33',
+    id: '33',
+    name: 'workflow_repository_test',
+    scm_branch: '',
+    scm_url: 'github.com:agrare/provision-vm-service',
```

Comment:
@agrare We really need to get an "examples" repo 😆

Reply:
Just say the word and I'll transfer :wink:

Reply:
@GilbertCherrie Please use `github.com:ManageIQ/workflows-examples`